src/lib/common/services/utils/utils.service.ts

```diff
@@ -8,6 +8,7 @@ import {IDate} from '../../models/date.model';
 import {CalendarMode} from '../../types/calendar-mode';
 import {DateValidator} from '../../types/validator.type';
 import {ICalendarInternal} from '../../models/calendar.model';
+import { IDayCalendarConfigInternal } from '../../../day-calendar/day-calendar-config.model';
```

Comment:
please remove spaces after/before the braces

Reply:
Actually, this import was unused, deleted